src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1014,7 +1014,14 @@ private void AppendParameters(SeparatedSyntaxList<ParameterSyntax> parameters, S
                     builder.Append(' ');
                 }
 
-                AppendTokens(parameter.Type, builder);
+                if (parameter.Type != null)
+                {
+                    AppendTokens(parameter.Type, builder);
+                }
+                else
+                {
+                    builder.Append(parameter.Identifier.Text);
```

Comment:
I'm using text here so that we just show users the results in NavTo that correspond to what they wrote in source.